Cilicon/Config/DirectoryMountConfig.swift

```diff
@@ -5,15 +5,19 @@ struct DirectoryMountConfig: Decodable {
     let hostPath: String
     /// The folder name in /Volumes/My Shared Files/ that the directory should be mounted to.
     let guestFolder: String
+    /// Mount the folder as a read only volume.
+    let readOnly: Bool
     
     init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         self.hostPath = (try container.decode(String.self, forKey: .hostPath) as NSString).standardizingPath
         self.guestFolder = try container.decode(String.self, forKey: .guestFolder)
+        self.readOnly = try container.decodeIfPresent(Bool.self, forKey: .readOnly) ?? .init()
```

Comment:
Could we fallback to `false` instead of `.init()`? The value of Bool after being initialized without an input is not specified anywhere.